examples/readme.md

```diff
@@ -0,0 +1,58 @@
+<img src="https://www.comet.ml/images/logo_comet_light.png" width="350" alt="Drawing" style="width: 350px;"/>
+
+<p align="left">
+    <a href="https://pypi.org/project/comet-llm">
+        <img src="https://img.shields.io/pypi/v/comet-llm" alt="PyPI version"></a>
+    <a rel="nofollow" href="https://opensource.org/license/mit/">
+        <img alt="GitHub" src="https://img.shields.io/badge/License-MIT-blue.svg"></a>
+    <a href="https://www.comet.com/docs/v2/guides/large-language-models/overview/" rel="nofollow">
+        <img src="https://img.shields.io/badge/cometLLM-Docs-blue.svg" alt="cometLLM Documentation"></a>
+    <a rel="nofollow" href="https://pepy.tech/project/comet-llm">
+        <img style="max-width: 100%;" src="https://static.pepy.tech/badge/comet-llm" alt="Downloads"></a>
+    <a rel="nofollow" href="https://colab.research.google.com/github/comet-ml/comet-llm/blob/main/examples/CometLLM_Prompts.ipynb">
+        <img src="https://colab.research.google.com/assets/colab-badge.svg"></a>
+</p>
+<p align="left">
+
+## CometLLM - Prompt and chain management
+**Our Misson:** CometLLM is a new suite of LLMOps tools designed to help you effortlessly track and visualize your LLM prompts and chains. Use CometLLM to identify effective prompt strategies, streamline your troubleshooting, and ensure reproducible workflows.
+
+CometLLM complements Comet experiment tracking and production model management tools to arm LLM practitioners with everything they need to interact with, manage, and optimize their models with ease.
+
+
+## Examples Repository
+
+This repository contains examples of using CometLLM to track, organize, and debug your LLM projects, including prompts and chains.
+
+If you don't see something you need, just let us know! See contact methods below.
+
+## Documentation
+[![PyPI version](https://badge.fury.io/py/comet-ml.svg)](https://badge.fury.io/py/comet-ml)
```

Comment:
This returns the version for the `comet-ml` package, not the `comet-llm` package, should be `https://badge.fury.io/py/comet-llm.svg` and `https://badge.fury.io/py/comet-llm`

Reply:
Changed!